docs/installation.md

```diff
@@ -72,7 +72,7 @@ See the [code style](code_style.md#ui) section for more details.
 
 ### Running the unit tests
 
-The unit tests for the UI are run with [Jest] and [Jasmine]. React components are tested with [enzyme].
+The unit tests for the UI are run with [Jest]. React components are tested with [enzyme].
```

Comment:
We might mention ``react-testing-library`` along side ``Enzyme``.

Reply:
And perhaps would it that we're migrating from `Enzyme` to `react-testing-library`?

Reply:
@camd, @sarah-clements - Thinking about this, it's probably best if the "Running the unit tests" section didn't mention anything other than Jest. The fact that the tests may or may not use jasmine/enzyme/react-testing-library is an implementation detail that isn't relevant to a beginner wondering how to run the tests.

Instead the code style section perhaps seems best for this. Can I leave it as a TODO for one of you to update that section later?

Reply:
Yeah, that makes sense.  Not mentioning the secondary testing library works for me.  Mentioning that our unit tests are ``Jest`` based is really sufficient.  :)

Reply:
Sure